### PR TITLE
Fix issue encountered when trying to reopen a file stored  remotely

### PR DESCRIPTION
**Pull request by zog**
When one tries to reopen a fog-stored file after it has been persisted, they get an error:

>  Magick::ImageMagickError: unable to open image

This is because the RMagick processor tries to open the file from its `current_path`, which in this case is the path on the remote server.
This is easily fixed by letting the uploader get the content og the image and send it to `  ::Magick::Image.from_blob`

### CODE REVIEW

**Comment by thiagofm** [comment]
Can you please add tests?

**Comment by zog** [comment]
There you are.

File: spec/processing/rmagick_spec.rb [hunk 1]
```diff
@@ -220,6 +220,38 @@
     end
   end
 
+  describe "#rmagick_image", focus: true do
```
Comment (mshibuya) [inline spec/processing/rmagick_spec.rb]: `focus: true` needs to be removed.

Comment (zog) [inline spec/processing/rmagick_spec.rb]: Oops :)

**Comment by mshibuya** [comment]
Nice, thanks!

**Pull request merged**